src/node.cc

```diff
@@ -1166,11 +1166,8 @@ void LoadEnvironment(Environment* env) {
       loader_exports.ToLocalChecked(),
       Boolean::New(isolate, env->is_main_thread())};
 
-  if (ExecuteBootstrapper(
-          env, "internal/bootstrap/node", &node_params, &node_args)
-          .IsEmpty()) {
-    return;
-  }
+  ExecuteBootstrapper(
+      env, "internal/bootstrap/node", &node_params, &node_args);
```

Comment:
I think this can all fit on one line now?

Reply:
Thank you for pointing that out. I updated PR.